dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-bitbucket-cloud/dist-dynamic/package.json

```diff
@@ -12,7 +12,9 @@
   },
   "backstage": {
     "role": "backend-plugin-module",
-    "supported-versions": "1.26.5"
+    "supported-versions": "1.27.7",
```

Comment:
this should move to 1.29.latest, right? If you prefer I can do that with a sed PR after this is merged.